crates/sequencer/Cargo.toml

```diff
@@ -27,7 +27,6 @@ tracing.workspace = true
 strata-test-utils.workspace = true
 
 [lints]
-rust.missing_docs = "warn"
```

Comment:
Please don't remove this lint? Use `clippy::allow` instead.

Reply:
it's a little excessive